src/simd/distances_avx.cc

```diff
@@ -38,14 +40,38 @@ masked_read(int d, const float* x) {
     // cannot use AVX2 _mm_mask_set1_epi32
 }
 
+namespace {
+
+inline __m256
+avx2_bf16_patch(__m256&& f256) {
```

Comment:
Why && here? Just passing by value seems better, cause the compiler can optimize it better